modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -117,8 +117,9 @@ For a detailed description of the configuration parameters, see the <<cloudhub2-
 If not set, defaults to +https://anypoint.mulesoft.com+. | No
 | `muleVersion` | The Mule runtime engine version to run in your CloudHub 2.0 instance.  +
 Ensure that this value is equal to or higher than the earliest required Mule version of your application. By default, the latest available Mule version is selected. +
-Example values: `4.6.0` selects the last build where it defaults to Edge channel and Java 8. If you want to select a different channel or Java version, specify the full Mule version name as in `4.6.0:1e-java17`. +
-When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, the `muleVersion` property doesn't allow you to specify the release channel and Java version which may cause unexpected behavior. To specify the release channel and Java version, upgrade the plugin to version 4.1.0. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes]. | No
+Example values: `4.6.0` selects the last build where it defaults to Edge channel and Java 8. If you want to select a different LTS or Edge selection or Java version, specify the full Mule version name as in `4.6.0:1e-java17`. +
+When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, the `muleVersion` property doesn't allow you to specify `releaseChannel` and `javaVersion`. To specify these properties, upgrade the plugin to version 4.1.1 or later. +
+Because Mule runtime 4.4 Standard Support ends in October 2024, new Mule Apps can be deployed on either Mule 4.6 LTS or Mule 4.8 Edge. As LTS channel is not supported for customers using a Mule Maven plugin version 4.1.0 or earlier, upgrade the plugin to version 4.1.1 or later. | No
```

Comment:
```suggestion
Because Mule runtime 4.4 Standard Support ends in October 2024, new Mule apps can be deployed on either Mule 4.6 LTS or Mule 4.8 Edge. As LTS channel is not supported for customers using a Mule Maven plugin version 4.1.0 or earlier, upgrade the plugin to version 4.1.1 or later. | No
```